### PR TITLE
PWA-9 remove i18n

### PR DESCRIPTION
**Pull request by jeffgo10**
remove i18n calls and other related matters from api-manager

### CODE REVIEW

File: packages/entando-apimanager/modules/api/apiManager.js [hunk 1]
```diff
@@ -146,7 +145,7 @@ export const makeRealRequest = (request, page) => {
   validateRequest(request);
   if (request.useAuthentication && !getAuthenticationToken()) {
     store.dispatch(logoutUser());
-    return new Promise(resolve => resolve({ ok: false, status: 401 }));
+    return Promise.reject(new Error('app.permissionDenied'));
```
Comment (gidesan) [inline packages/entando-apimanager/modules/api/apiManager.js]: why this change? @nicpuddu I guess lots of thunks rely on a resolved promise with 401 status

File: packages/entando-apimanager/modules/api/apiManager.js [hunk 2]
```diff
@@ -168,19 +167,9 @@ export const makeRealRequest = (request, page) => {
...
-        { domain: getDomain(store.getState()) },
-      ),
-      TOAST_ERROR,
-    ));
-    // eslint-disable-next-line no-console
-    console.error(e);
-    throw e;
-  });
+  }).catch(e => (
+    Promise.reject(new Error(`app.${normalizeErrorMessage(e.message)}`))
```
Comment (gidesan) [inline packages/entando-apimanager/modules/api/apiManager.js]: The info about domain is lost here. Do we think is useless?

File: packages/entando-apimanager/modules/api/apiManager.js (hunk 1 above)
Comment (nicpuddu) [reply packages/entando-apimanager/modules/api/apiManager.js]: thing is reject throws an error, and it should be returned only if the request fails horribly, so yeah I agree with @gidesan

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: okay. I have restored the resolve promise indicated

File: packages/entando-apimanager/modules/api/apiManager.js (hunk 2 above)
Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: it was supposed to use only info from `process.env.DOMAIN` in app builder but anyway I have implemented back the `domain` but this time injected in `error.data`.

Comment (gidesan) [reply packages/entando-apimanager/modules/api/apiManager.js]: Doing this way, we suppose on the other side someone should know the Error object has this added data field.
What about passing a custom error containing a message descriptor with message, defaultMessage and parameters? @nicpuddu

Comment (nicpuddu) [reply packages/entando-apimanager/modules/api/apiManager.js]: sounds like a good idea @gidesan

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: okay guys, I'll make a custom error for this then ;)

Comment (nicpuddu) [reply packages/entando-apimanager/modules/api/apiManager.js]: @jeffgo10 I'd just like to add that to me it is way more sensible if the `apimanager` exports among other things a json compliant with the app-builder i18n so that we are not forced to implement those translations from scratch.

see the locales files inside app-builder itself.

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: @nicpuddu sorry i'm not sure if I get what you mean, but do you meant not to provide defaultmessage for the Errors inside apimanager or we provide a json format locale library for the message descriptors inside apimanager? What I just did for now is just to decamelize the message ids and make them as default messages. Let me know what you guys think

Comment (nicpuddu) [reply packages/entando-apimanager/modules/api/apiManager.js]: the module itself should just be exporting a message file akin to the ones in app builder, so that appbuilder can just import them and merge it with theirs instead of having to deal with resetting the messages on its own.

just like we used to do for plugins.

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: okay @nicpuddu. Thanks for the suggestion. I'll work on that.

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: Hi @nicpuddu. I have added `locales` as an export and also used it as default message for the `ErrorI18n` error dispatch. Please review

Comment (gidesan) [reply packages/entando-apimanager/modules/api/apiManager.js]: @jeffgo10 can you please capitalize the first letter of the translated messages? :)

Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: @gidesan done :)
